qemu/tests/cfg/data_plane_boot.cfg

```diff
@@ -1,7 +1,5 @@
-- data_plane_snapshot:
+- data_plane_boot:
     virt_test_type = qemu
-    type = boot
-    image_snapshot = yes
     only Host_RHEL.m7
```

Comment:
In order to cover ALT host,  should use "no Host_RHEL.m6" here.

Reply:
Both RHEL6 and RHEL5 should be excluded.

Reply:
I don't think it is very necessary as avocado doesn't even support RHEL6 officially, but It is ok for me to exclude RHEL 5 too. 
My option is to support new version.

Reply:
@yanan-fu,confirmed with hanxu，avocado not support RHEL5，so just set the the filter to "no Host_RHEL.m6", thanks for pointing it out.